plugins/recent-doc/Cargo.toml

```
@@ -0,0 +1,43 @@
+[package]
+name = "tauri-plugin-recent-doc"
+version = "1.0.0"
+description = "Manage recent documents in your Tauri application."
+edition = { workspace = true }
+authors = { workspace = true }
+license = { workspace = true }
+rust-version = { workspace = true }
+repository = { workspace = true }
+links = "tauri-plugin-recent-doc"
+
+# Platforms supported by the plugin
+# Support levels are "full", "partial", "none", "unknown"
+# Details of the support level are left to plugin maintainer
+[package.metadata.platforms]
+windows = { level = "full", notes = "" }
+linux = { level = "none", notes = "" }
+macos = { level = "full", notes = "" }
+android = { level = "none", notes = "" }
+ios = { level = "none", notes = "" }
+
+
+[build-dependencies]
+tauri-plugin = { workspace = true, features = ["build"] }
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { workspace = true }
+serde_json = { workspace = true }
+tauri = { workspace = true }
+log = { workspace = true }
+thiserror = { workspace = true }
+
+[target."cfg(target_os = \"macos\")".dependencies]
+objc2 = "0.6.0"
+
+[target."cfg(target_os = \"windows\")".dependencies]
+windows = { version = "0.62.2", features = [
```

```suggestion
windows = { version = "0.61", features = [
```

We need to use 0.61 here for the lower rust version requirement to support Windows 7 in tauri v2